kubernetes/example/deployment/kylin-job/kylin-job-service.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kylin-job-svc
```

Comment:
In my test, if I use only one service, the SQL query request will sent to job server, and the request will be rejected because job server won't answer such request.

Reply:
Yes, that’s why I suggested to remove the service. It is unnecessary. And user talk with Kylin by query service, even call build job api.

Reply:
OK, I finally understand what you mean, thanks for this suggestion, it do make sense.